cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/schedule/queue/SimpleJob.java

```diff
@@ -31,12 +31,16 @@
   private final JobKey jobKey;
   private final List<Notification> notifications;
   private final State state;
+  private final long scheduleTimestamp;
```

Comment:
I think the variable name should be consistent with the getter.
Getter implies that its a last updated timestamp, but variable name here (and also the constructor parameter) don't give that implication.